org.lflang/src/org/lflang/generator/python/PythonGenerator.java

```diff
@@ -1,5 +1,3 @@
-/* Generator for the Python target. */
```

Comment:
This change silences a warning. Perhaps comments of this sort should be discouraged (their content is obvious given the class name)